### PR TITLE
Add clear featured

### PR DESCRIPTION
**Pull request by SamamthaH**
In Views and Services, I added the code required for clearing the featured property from an event and for the front end to access that code.

In Models, I added code to clear the featured property when an event closes.

### CODE REVIEW

File: canvas/services/event.py [hunk 1]
```diff
@@ -73,6 +73,12 @@ def set_featured(event):
     event.save()
 
 
+def clear_featured(event):
+    event.course.events.update(featured=False)
```
Comment (keyvankhademi) [inline canvas/services/event.py]: This will remove all the features. Probably you don't want it here.

File: canvas/models/models.py [hunk 2]
```diff
@@ -238,6 +238,8 @@ def is_open(self):
 
     @property
     def is_closed(self):
+        if self.featured & (self.end_date < timezone.now()):
+            self.featured = False
```
Comment (keyvankhademi) [inline canvas/models/models.py]: This doesn't do what you intended to. You are changing the property on a property (it's equivalent to changing a class variable in a getter method in java). Also, it doesn't actually save the model.
Let's remove this for this PR and if you want to clear the feature of a closed assignment automatically do it properly in another PR.

Comment (SamamthaH) [reply canvas/models/models.py]: where would be the proper place to do it? I couldn't find any other place that seemed to be where the closed vs open vs not yet available were calculated/moved between

**Review by keyvankhademi — APPROVED** [review]

**Review by keyvankhademi — APPROVED** [review]
